### PR TITLE
Carson/simpler onboarding

### PR DESCRIPTION
**Pull request by carsonfarmer**
Not all that happy with this, but should be GTG if someone wants to make some comments and maybe merge. I'll try to find a nicer way to do the terms of service link

### CODE REVIEW

File: App/Navigation/PhotosNavigation.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ const TabNav = TabNavigator(
     tabBarComponent: TabBarBottom,
     tabBarPosition: 'bottom',
     animationEnabled: false,
-    swipeEnabled: false
```
Comment (asutula) [inline App/Navigation/PhotosNavigation.js]: I had specifically set this to false. Any reason it needs to be true?

File: App/Navigation/OnboardingNavigation.js [hunk 2]
```diff
@@ -1,78 +1,51 @@
...
-      navigate('Onboarding4', params6)
-    }
-  }
-}
-
-const params6 = {
   header: 'You\'re helping us improve.',
-  message: 'As a valued Beta Tester, we want to know how, when, and why you are using the app. We anonymously collect data, including crashes, screen interactions, and feedback. If you want to provide direct feedback, please feel free to email us with you thoughts, comments, or ideas.',
+  message: (
+    <View>
```
Comment (asutula) [inline App/Navigation/OnboardingNavigation.js]: `OnboardingScreen` currently is wrapping the value if this `message` prop in a pair of `<Text>` tags. So in this case, it's rendering `<Text><View>...</View></Text>`, which might work, but is not right. I'd just update `OnboardingScreen` to expect that the value of message is a component without assuming what it is exactly. Know what I mean?

File: App/Navigation/PhotosNavigation.js (hunk 1 above)
Comment (carsonfarmer) [reply App/Navigation/PhotosNavigation.js]: oh sorry, that was a mistake... was playing around and didn't set it back. will revert

**Comment by carsonfarmer** [comment]
yup i'll update those two things and push @asutula

File: App/Navigation/OnboardingNavigation.js (hunk 2 above)
Comment (carsonfarmer) [reply App/Navigation/OnboardingNavigation.js]: fixed

**Comment by carsonfarmer** [comment]
GTG @asutula

**Pull request merged**